api/src/main/java/org/motechproject/nms/api/web/validator/MobileAcademyValidator.java

```diff
@@ -0,0 +1,114 @@
+package org.motechproject.nms.api.web.validator;
+
+import org.motechproject.nms.api.web.contract.mobileAcademy.CourseResponse;
+import org.motechproject.nms.api.web.contract.mobileAcademy.course.Chapter;
+import org.motechproject.nms.api.web.contract.mobileAcademy.course.Lesson;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.Validation;
+import javax.validation.Validator;
+import java.util.Set;
+
+
+/**
+ * Validator helper class for API request and response
+ */
+public final class MobileAcademyValidator {
+
+    private static final String CHAPTER_NAME_FORMAT = "Chapter<%d>";
+
+    private static final String CHAPTER_END_MENU_FORMAT = "Chapter<%d>_EndMenu";
+
+    private static final String CHAPTER_SCORE_FORMAT = "Chapter<%d>_Score";
+
+    private static final String LESSON_NAME_FORMAT = "Lesson<%d>";
+
+    private static final String LESSON_CONTENT_ID_FORMAT = "Chapter<%d>_Lesson<%d>";
+
+    private static final String LESSON_CONTENT_MENU_ID_FORMAT = "Chapter<%d>_LessonEndMenu<LessonId>";
+
+    private static final String FORMAT_ERROR = " does not meet format requirements ";
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(MobileAcademyValidator.class);
+
+    /**
+     * Private constructor for static validation helpers
+     */
+    private MobileAcademyValidator() {
+
+    }
+
+    public static String validateCourseResponse(CourseResponse courseResponse) {
+        Validator validator = Validation.buildDefaultValidatorFactory().getValidator();
+        Set<ConstraintViolation<CourseResponse>> violations = validator.validate(courseResponse);
+        StringBuilder sb = new StringBuilder();
+
+        if (violations.size() > 0) {
+            for (ConstraintViolation<CourseResponse> violation : violations) {
+                sb.append(violation.getInvalidValue() + " : " +
+                        violation.getPropertyPath() + " - " + violation.getMessage());
+                sb.append("\n");
+            }
+
+            return sb.toString();
+        } else {
+            return null;
+        }
+    }
+
+    public static boolean validateCourseFormat(CourseResponse courseResponse) {
+
+        int i = 1;
```

Comment:
maybe name the `i` & `l` variables something related to what they're supposed to represent?